R/dataframe__frame.R

```diff
@@ -1640,9 +1640,6 @@ DataFrame_glimpse = function(..., return_as_string = FALSE) {
 #' # explode a single column, append others
 #' df$explode("numbers")
 #'
-#' # it doesn't change anything if the input is not a list-column
-#' df$explode("letters")
```

Comment:
I think this surprising behavior should be included in the example.

Reply:
done